### PR TITLE
Remove custom repo step from setup instructions

### PR DESCRIPTION
**Pull request by ThomDietrich**
Follow up after #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![hacs_badge](https://img.shields.io/badge/HACS-Custom-41BDF5.svg)](https://github.com/hacs/integration)
+[![hacs_badge](https://img.shields.io/badge/HACS-Default-41BDF5.svg?style=for-the-badge)](https://github.com/hacs/integration)
```
Comment (ThomDietrich) [inline README.md:1]: Good catch :)

**Comment by ThomDietrich** [comment]
One more step added:

- https://hacs.xyz/docs/basic/my/

**Comment by ThomDietrich** [comment]
Within HACS images are not shown. @edwardtfn do you have the same issue?

image

**Comment by edwardtfn** [comment]
Yes, same issue here. ☹️

**Pull request merged**